pom.xml

```diff
@@ -56,7 +56,7 @@
     <version.checksum-maven-plugin>1.11</version.checksum-maven-plugin>
     <awaitility.version>4.2.0</awaitility.version>
     <checkstyle.version>9.3</checkstyle.version>
-    <commons-collections.version>3.2.2</commons-collections.version>
+    <commons-collections4.version>4.5.0-M1</commons-collections4.version>
```

Comment:
I suggest migrating to 4.4 first, rather than a milestone version

Reply:
Done.